docs/using.md

```diff
@@ -5,86 +5,56 @@ overview of different components and services.
+- [HTTP/HTTPS Proxies](#httphttps-proxies)
+- [Mounted host folders](#mounted-host-folders)
+  - [Mounting custom shared folders](#mounting-custom-shared-folders)
+- [Networking](#networking)
 - [Interacting with OpenShift](#interacting-with-openshift)
   - [OpenShift client binary \(oc\)](#openshift-client-binary-oc)
   - [Login](#login)
   - [Console](#console)
   - [Services](#services)
-- [HTTP/HTTPS Proxies](#httphttps-proxies)
-- [Mounted host folders](#mounted-host-folders)
-  - [Mounting custom shared folders](#mounting-custom-shared-folders)
-- [Networking](#networking)
-- [Persistent volumes](#persistent-volumes)
-- [Private container registries](#private-container-registries)
 
 <!-- /MarkdownTOC -->
 
-<a name="managing-your-openshift-instance"></a>
-## Managing your OpenShift instance
+<a name="managing-minishift"></a>
+## Managing Minishift
+When you use Minishift, you actually interact with:
+- the virtual machine(VM) created by Minishift and
+- the OpenShift instance provisioned by the VM.
 
-This section contains information about basic virtual machine and OpenShift management operations.
+This section contains information about managing the Minishift VM. For details on managing your local OpenShift instance using Minishift refer to the [Interacting with OpenShift](#interacting-with-openshift) and the [Configuring OpenShift](./configuring-openshift.md) sections.
 
-<a name="starting-openshift"></a>
-### Starting OpenShift
+<a name="minishift-start"></a>
+### Minishift start
 
-The [`minishift start`](./minishift_start.md) command is used to start your OpenShift instance.
-This command creates and configures a virtual machine that runs a single-node OpenShift instance.
+The [`minishift start`](./minishift_start.md) command is used to start a local OpenShift instance.
+This command creates and configures the Minishift VM that runs a single-node OpenShift instance.
 
-<a name="stopping-openshift"></a>
-### Stopping OpenShift
+It also copies the `oc` binary to your host so that you can interact with OpenShift either through the `oc` command line tool or through the web console which can be accessed through the URL provided in the output of the `minishift start` command. For further information refer to the [Interacting with OpenShift](#interacting-with-openshift) and [Configuring OpenShift](./configuring-openshift.md) sections.
+
+<a name="minishift-stop"></a>
+### Minishift stop
 
 The [`minishift stop`](./minishift_stop.md) command is used to stop your OpenShift instance.
-This command shuts down the Minishift virtual machine, but preserves the cluster state.
-Starting Minishift again will restore the cluster, allowing you to continue work from where you left-off.
+This command shuts down the Minishift VM, but preserves the OpenShift cluster state.
+Starting Minishift again will restore the OpenShift cluster, allowing you to continue work from where you left-off.
```

Comment:
Let's add a note here, that on re-start one needs to repeat the same parameters used in the original start command. This is the most common usage issue we have atm. We can also add a reference to issue #179